docs/CLI.md

```diff
@@ -61,3 +61,12 @@ gulp has very few flags to know about. All other flags are for tasks to use if n
 
 **--log-level**, **-L**
     Set the loglevel. -L for least verbose and -LLLL for most verbose. -LLL is default.
+
+**--config**, **-c**
```

Comment:
This flag is useful for using gulp while switching some configuration files, for applying some configurations temporarily, for testing configuration files, etc.

These three flags above are not necessary until someone needs them, as you said. We can remove these features because it is easy to add them again.